### ISSUE TITLE
Amend 'Accessibility' link footer and page title

### ISSUE THREAD

**Issue by dgamblin3008** [DFE-Digital/npd-find-and-explore#620]
**INFO**
1. GDS guidance states that links to Accessibility statements on GOV.UK should be called 'Accessibility statement', as is shown on the GOV.UK website itself, here:
![Accessibility statement link 25-09-2020](https://user-images.githubusercontent.com/66372153/94256325-8f376f80-ff21-11ea-90a3-cb3e598c6a92.PNG)
2. The guidance states that the Accessibility statement page should have the title 'Accessibility statement for [website name]'. The guidance is here https://www.gov.uk/government/publications/sample-accessibility-statement/sample-accessibility-statement-for-a-fictional-public-sector-website
3. Other DfE websites, e.g. CSCP, GIAS, ASP, are in the process of changing to this format.

**ACCEPTANCE CRITERIA**
1. The link to the accessibility statement page that is at the bottom of all NPD Find & Explore pages is changed from 'Accessibility' to 'Accessibility statement'.
2. On the accessibility statement page, the title of the page is changed from 'Accessibility' to 'Accessibility statement for Find and Explore Data in the National Pupil Database'.
3. On the accessibility statement page, the sub-heading text that reads 'Accessibility for Find and Explore data in the National Pupil Database' is removed.

**Comment by GMcGill-DfE** [comment]
Approved

**Comment by dgamblin3008** [comment]
Assigning to @cacquani to action.

**Comment by dgamblin3008** [comment]
Tested and released to live on 07/10/2020. ticket closed.
